### ISSUE TITLE
Applying a profile while in DND mode causes SecurityException

### ISSUE THREAD

**Issue by d4rken** [punksta/volume_control_android#5]
Pixel2@Android9

App version: `2.1.0`

Put phone into DND mode, create profile, save, click apply.

```
2019-06-21 20:33:33.242 31058-31058/? E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.punksta.apps.volumecontrol, PID: 31058
    java.lang.RuntimeException: Unable to start service com.example.punksta.volumecontrol.SoundService@f538380 with Intent { act=APPLY_PROFILE cmp=com.punksta.apps.volumecontrol/com.example.punksta.volumecontrol.SoundService (has extras) }: java.lang.SecurityException: Not allowed to change Do Not Disturb state
        at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:3724)
        at android.app.ActivityThread.access$1600(ActivityThread.java:200)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1688)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6718)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
     Caused by: java.lang.SecurityException: Not allowed to change Do Not Disturb state
        at android.os.Parcel.createException(Parcel.java:1950)
        at android.os.Parcel.readException(Parcel.java:1918)
        at android.os.Parcel.readException(Parcel.java:1868)
        at android.media.IAudioService$Stub$Proxy.setStreamVolume(IAudioService.java:1132)
        at android.media.AudioManager.setStreamVolume(AudioManager.java:1128)
        at a.a.a.a.c.a(Unknown Source:3)
        at com.example.punksta.volumecontrol.b.c.a(Unknown Source:42)
        at com.example.punksta.volumecontrol.SoundService.onStartCommand(Unknown Source:31)
        at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:3705)
        at android.app.ActivityThread.access$1600(ActivityThread.java:200) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1688) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loop(Looper.java:193) 
        at android.app.ActivityThread.main(ActivityThread.java:6718) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858) 
     Caused by: android.os.RemoteException: Remote stack trace:
        at com.android.server.audio.AudioService.setStreamVolume(AudioService.java:1974)
        at com.android.server.audio.AudioService.setStreamVolume(AudioService.java:1916)
        at android.media.IAudioService$Stub.onTransact(IAudioService.java:139)
        at android.os.Binder.execTransact(Binder.java:731)
```

**Comment by punksta** [comment]
Good catch! will be fixed in next version

**Comment by punksta** [comment]
as a workaround try to use any volume slider before applying profiles. You will be asked to grant a DND change permission.

**Comment by punksta** [comment]
fixed in 2.2.0
